app/src/docs/content/introduction/introduction.mdx

```diff
@@ -44,19 +44,19 @@ It is similar in functionality to message queues, enterprise messaging systems,
 In addition to the Network, the Streamr stack includes a <Link to={docsLinks.marketplace}>Marketplace</Link> for data sharing and monetization, and <Link to={docsLinks.canvases}>Canvases</Link> for data processing and analytics as part of the Core web app.
 
-Using the built-in tooling is optional, as Streamr also integrates with many popular data frameworks including <a target="_blank" href={routes.blogPosts.spark()}>Apache Spark</a> and <a target="_blank" href={routes.blogPosts.nodered()}>Node-RED</a>. To start using Streamr right away, check out the <Link to={docsLinks.gettingStarted}>Getting Started</Link> section and the <Link to={docsLinks.tutorials}>Tutorials.</Link>
+Using the built-in tooling is optional, as Streamr also integrates with many popular data frameworks including <a target="_blank" rel="noopener noreferrer" href={routes.blogPosts.spark()}>Apache Spark</a> and <a target="_blank" rel="noopener noreferrer" href={routes.blogPosts.nodered()}>Node-RED</a>. To start using Streamr right away, check out the <Link to={docsLinks.gettingStarted}>Getting Started</Link> section and the <Link to={docsLinks.tutorials}>Tutorials.</Link>
 
 ## Why use Streamr?
 The big difference to comparable data frameworks is that Streamr aims to be decentralized and peer-to-peer. There are plenty of advantages to decentralization.
 
 Similar to cloud services, you get on-demand scalability, minimal up-front investment, and benefit from economies of scale. Unlike cloud services, there's no vendor lock-in, no monopolies, no proprietary code, and no need to trust a third party with your data. Decentralised networks are also secure by design, more resilient to attacks and disruptions due to their distributed and peer-to-peer nature. They also benefit from network effects— as adoption grows, the above points become stronger, a shared standard is formed across industries, and a lot of useful content becomes available from a single source.
 
-Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for real-time data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" href={routes.misc.decentralizedWeb()}>decentralised web</a>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
+Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for real-time data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" rel="noopener noreferrer" href={routes.misc.decentralizedWeb()}>decentralised web</a>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
```

Comment:
Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for real-time data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" rel="noopener noreferrer" href={routes.misc.decentralizedWeb()}>decentralised web</a>, extending the capabilities of the underlying internet without compromising its open and decentralised nature.
```suggestion
Instead of a centralized party, the Network is run by its community of users, and it heavily relies on cryptography to remove the need for trust. This creates a permissionless and neutral network for real-time data, which is highly suitable for use cases such as data sharing, data monetisation, and the <a target="_blank" rel="noopener noreferrer" href={routes.misc.decentralizedWeb()}>decentralized web</a>, extending the capabilities of the underlying internet without compromising its open and decentralized nature.
```